Krypt/Source/helper.c

```diff
@@ -0,0 +1,55 @@
+//
+//  helper.c
+//  Krypt
+//
+//  Created by Miso Lubarda on 25.04.19.
+//
+
+#include "helper.h"
+#include <string.h>
+#include <openssl/buffer.h>
+
+
+/**
+ Copies buffer to string, adding NULL termination to string
+
+ @param mem Buffer to copy from
+ @return String from the content of buffer
+ */
+char *copy_mem_buf(BUF_MEM* mem) {
+  char *str = malloc(mem->length + 1);
+  memcpy(str, mem->data, mem->length + 1);
```

Comment:
It looks like only amount of mem->length need to be copied.
Like this?
```suggestion
  memcpy(str, mem->data, mem->length);
```

Reply:
Thanks for pointing this out!